### PR TITLE
Compact measurement API serializer

### PR DESCRIPTION
**Pull request by jimr**

### CODE REVIEW

File: observations/api/views.py [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
+    CompactMeasurementSerializer, MeasurementSerializer,
+)
 
 
 class MeasurementList(generics.ListAPIView):
-    serializer_class = MeasurementSerializer
     queryset = Measurement.objects.all()
+
+    def get_serializer_class(self):
+        if self.request.QUERY_PARAMS.get('compact'):
```
Comment (stringfellow) [inline observations/api/views.py]: Nice! How about https://docs.djangoproject.com/en/dev/ref/request-response/#django.http.HttpRequest.is_ajax instead? Then we don't have to document options!

**Comment by jimr** [comment]
You mean default to compact for XHR?

**Comment by stringfellow** [comment]
Yeah, basically. What do you reckon?

**Comment by jimr** [comment]
Makes sense I suppose, aye.

**Comment by jimr** [comment]
Boom, there you go :)

**Pull request merged**